# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1236232556 18000
#      Thu Mar 05 00:55:56 2009 -0500
# Node ID 52596fe9f17806840d5a8d06448a187de78efb54
# Parent  3c9acbb43f4836a63ef50de0d7dbb224eebcdbf9
__go_draw_axes__.m: Preserve column vector orientation for the axes children while removing the axis label & title handles.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2009-03-05  Ben Abbott <bpabbott@mac.com>
+
+	* plot/__go_draw_axes__.m: Preserve column vector orientation for
+	the axes children while removing the axis label & title handles.
+
 2009-03-03  Jaroslav Hajek  <highegg@gmail.com>
 
 	* polynomial/polyval.m: Implement using Horner scheme.
 
 2009-03-03  Ben Abbott <bpabbott@mac.com>
 
 	* plot/gnuplot_drawnow.m: Fix unintended shift of plot image for
 	the gnuplot postscript+eps terminal.
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -289,18 +289,18 @@ function __go_draw_axes__ (h, plot_strea
     zautoscale = strcmpi (axis_obj.zlimmode, "auto");
     cautoscale = strcmpi (axis_obj.climmode, "auto");
     cdatadirect = false;
     truecolor = false;
 
     fputs (plot_stream, "set clip two;\n");
 
     kids = axis_obj.children;
-    kids = setdiff (kids, [axis_obj.xlabel, axis_obj.ylabel, ...
-                           axis_obj.zlabel, axis_obj.title]);
+    kids = setdiff (kids, [axis_obj.xlabel; axis_obj.ylabel; ...
+                           axis_obj.zlabel; axis_obj.title]);
 
     if (nd == 3)
       fputs (plot_stream, "set parametric;\n");
       fputs (plot_stream, "set style data lines;\n");
       fputs (plot_stream, "set surface;\n");
       fputs (plot_stream, "unset contour;\n");
     endif
 
